modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/settings.gradle

```diff
@@ -0,0 +1 @@
+include "${artifactId}-api", "${artifactId}-client", "${artifactId}-impl", "${artifactId}-test"
```

Comment:
We currently only support Project templates in 3 different builds
standalone maven
maven based Liferay Workspace
gradle based Liferay Workspace

We don't support standalone gradle anymore.